pos_self_service_weight_base/static/src/xml/Screens/SelfServiceWeightScreen/SelfServiceWelcomeScreen.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<templates id="template" xml:space="preserve">
+    <t t-name="pos_self_service_weight_base.SelfServiceWelcomeScreen" owl="1">
+        <div class="screen">
+            <div class="screen-full-width">
+                    <div class="self-service-action-buttons">
+                        <SelfServiceActionButton t-if="env.isDebug()" />
```

Comment:
why is this button shown only in debug mode?

Reply:
It exists to test the behavior of the base module when there are nos other modules installed. It can be removed in the future.